File: api/ZME.API.Shared/Models/Airport.cs
namespace ZME.API.Shared.Models;

public class Airport
{
    public int Id { get; set; }
    public required string Name { get; set; }
    public required string Icao { get; set; }
    public int Arrivals { get; set; }
    public int Departures { get; set; }
    public string? Wind { get; set; }
    public string? Altimeter { get; set; }
    public string? Temperature { get; set; }
    public string? MetarRaw { get; set; }
    public DateTimeOffset Created { get; set; } = DateTimeOffset.UtcNow;
    public DateTimeOffset Updated { get; set; } = DateTimeOffset.UtcNow;
}

File: ui/src/models/Airport.ts
export type Airport = {
    id?: string;
    name: string;
    icao: string;
    arrivals: number;
    departures: number;
    wind?: string;
    altimeter?: string;
    temperature?: string;
    metarRaw?: string;
    created?: Date;
    updated?: Date;
}